### PR TITLE
Alp/consume api

### PR DESCRIPTION
**Pull request by alpdurmaz**

### CODE REVIEW

**Comment by alpdurmaz** [comment]
In this work we are able to get tweets from api via TwitterSourceRepository class. 
TwitterConfigurer class provides connection to twitter api. 
Also TwitterSearchFailedException class created and test code written

**Comment by snaket2003** [comment]
Hey @alpdurmaz . Do you mind resolving the conflicts between this branch and merged master before the review? You will need to stash your changes, rebase, pop your changes and resolve any conflicts accordingly :/

File: src/main/java/com/codingnomads/betty/configurations/TwitterConfigurer.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        return twitter;
+    }
+
+    private void buildTwitterConfiguration() {
+
+        ConfigurationBuilder configurationBuilder = new ConfigurationBuilder();
+
+        configurationBuilder
+                .setDebugEnabled(true)
+                .setOAuthConsumerKey("hv8cRVlIIsTK44fly4Bdd6sRn")
```
Comment (snaket2003) [inline src/main/java/com/codingnomads/betty/configurations/TwitterConfigurer.java]: We need to find a way to not expose the keys on the code. Maybe use Travis or find a different way?

File: src/main/java/com/codingnomads/betty/logic/services/TwitterService.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package com.codingnomads.betty.logic.services;
+
+import com.codingnomads.betty.logic.repositories.TwitterRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import twitter4j.Status;
+import java.util.List;
+
+public class TwitterService {
```
Comment (snaket2003) [inline src/main/java/com/codingnomads/betty/logic/services/TwitterService.java]: I think this needs to be annotated with @Service?

File: src/main/java/com/codingnomads/betty/database/TwitterSourceRepository.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+import com.codingnomads.betty.configurations.TwitterConfigurer;
+import com.codingnomads.betty.logic.exceptions.TwitterSearchFailedException;
+import com.codingnomads.betty.logic.repositories.TwitterRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import twitter4j.*;
+import java.util.List;
+
+@Service
```
Comment (snaket2003) [inline src/main/java/com/codingnomads/betty/database/TwitterSourceRepository.java]: I think this needs to be annotated as a @Repository and not a service

**Pull request merged**